### PR TITLE
Add RHEL-9.4 downloaded package

### PR DESCRIPTION
**Pull request by riddhi-ibm**

### CODE REVIEW

**Comment by riddhi-ibm** [comment]
```
Debug - File unlocked:/var/lock/ibm_mount_helper.lck
Debug - RunCmd: MountCmd (mount -t nfs4 -o sec=sys,nfsvers=4.1,rw 10.240.64.4:/2895b288_a3d5_4a02_ad1b_83a13d5d8d11 /mnt/share-test -v)
Debug - RunCmd: LoadCert (openssl x509 -in /etc/strongswan/swanctl/x509ca/type_ibmshare_int.crt -noout -dates -subject -issuer)
Debug - RunCmd: LoadCert (openssl x509 -in /etc/strongswan/swanctl/x509ca/type_ibmshare_root_tor.crt -noout -dates -subject -issuer)
Share successfully mounted:mount.nfs4: timeout set for Thu Aug 22 16:05:00 2024
mount.nfs4: trying text-based options 'sec=sys,nfsvers=4.1,addr=10.240.64.4,clientaddr=10.240.0.8'
```

**Comment by riddhi-ibm** [comment]
_Installed okay_
```
Debug - ReadFile:./certs/metadata/type_ibmshare_root_par.crt
Debug - WriteFile:/etc/strongswan/swanctl/x509ca/type_ibmshare_root_par.crt
Debug - ReadFile:./certs/metadata/type_ibmshare_root_dal.crt
Debug - WriteFile:/etc/strongswan/swanctl/x509ca/type_ibmshare_root_dal.crt
Debug - ReadFile:./certs/metadata/type_ibmshare_root_tok.crt
Debug - WriteFile:/etc/strongswan/swanctl/x509ca/type_ibmshare_root_tok.crt
Debug - RunCmd: ReloadCerts (/usr/sbin/swanctl --load-creds)

IBM Mount Share Helper: Install completed ok.
```

**Review by anand-desai — APPROVED** [review]

**Review by saicharansunkara-ibm — APPROVED** [review]

File: mount-helper/install/install.sh [hunk 1]
```diff
@@ -303,7 +304,7 @@ _install_apps() {
             fi
             PACKAGE_DIR="packages/ubuntu/$VERSION"
             dpkg --force-all -i "$PACKAGE_DIR/$app"*
-        elif  is_linux LINUX_RED_HAT && ([[ $VERSION == 8.8 || $VERSION == 8.9 || $VERSION == 8.10 ]]); then
+        elif  is_linux LINUX_RED_HAT && ([[ -d "$DOWNLOADED_RHEL_PACKAGE_PATH"]]); then
```
Comment (prankulmahajan) [inline mount-helper/install/install.sh:307]: Getting error
```
./install.sh: line 307: syntax error in conditional expression: unexpected token `)'
```

```suggestion
        elif  is_linux LINUX_RED_HAT && ([[ -d "$DOWNLOADED_RHEL_PACKAGE_PATH" ]]); then
```

**Review by prankulmahajan — CHANGES_REQUESTED** [review]
After adding the suggested changes, the testing for IKS looks good.

Couple of suggestions for this PR,

1. Should other version supported version of RHEL 9 be added? Like 9.0/9.2?
2. Can you create github release via this PR itself?

**Comment by riddhi-ibm** [comment]
> * Should other version supported version of RHEL 9 be added? Like 9.0/9.2? - 9.0 image we dont have in cloud.ibm I cannot test that . Will do this after the priority items are taken care of?
> * Can you create github release via this PR itself? - _Added_
_Suggested changes added._

**Review by prankulmahajan — APPROVED** [review]
lgtm

**Pull request merged**